website/src/libs/api-v3/baseModel.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+import { uuid } from '../../../../common';
```

Comment:
I had the same idea but there are 2 modules, uuid and node-uuid with the first being a fork of the second and there are talks of merging the two so I didn't want to rely on one and then have to change it in 100 different places if it ever changed